packages/components/input-number/src/input-number.vue

```diff
@@ -196,6 +196,11 @@ const verifyValue = (
   update?: boolean
 ): number | null | undefined => {
   const { max, min, step, precision, stepStrictly, valueOnClear } = props
+  if (max < min) {
+    throw new Error(
+      'InputNumber The maximum value of a component is smaller than the minimum values'
+    )
+  }
```

Comment:
Maybe you should use this function
https://github.com/element-plus/element-plus/blob/213fdfb51e51ec8a29bba4317e29a2b2ea25aa3f/packages/utils/error.ts#L10-L12

Reply:
if (max < min) {
    debugWarn(
      'InputNumber',
      'The maximum value of a component is smaller than the minimum values'
    )
    return null
  }
Is this way of writing all right with you?

Reply:
IMO,  just replace it with `throwError` as written above. Maybe you can refer to here
https://github.com/element-plus/element-plus/blob/213fdfb51e51ec8a29bba4317e29a2b2ea25aa3f/packages/components/slider/src/composables/use-watch.ts#L33-L35